assets/src/edit-story/components/button/index.js

```diff
@@ -128,6 +128,25 @@ export const GridView = (props) => (
   </StyledButton>
 );
 
+export const Rectangle = styled(Base)`
+  border: 0;
+  padding: 0;
+  margin: 0;
+  min-width: unset;
+  height: auto;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  color: ${({ theme }) => theme.colors.fg.v1};
+  ${({ isLarge }) => (isLarge ? 'margin-left' : 'margin-right')}: 20px;
+
+  svg {
+    width: ${({ isLarge }) => (isLarge ? '20px' : '12px')};
+    height: auto;
+    shape-rendering: crispEdges; /* prevents issues with anti-aliasing */
+  }
+`;
```

Comment:
If this is a component made to specifically show the rectangle icon, it should just include the rectangle icon (be a "real" component). And if so, it doesn't really belong as a "generic" button as it probably won't be used anywhere else.

If it's supposed to be a generic "icon" button, it should be named as such. But we already have another generic icon button, as far as I know?

What is the intent?

Reply:
It's not a generic Icon

Reply:
The entire application has at least 30 different icon buttons throughout the UI. I don't think we need or want a separate definition for each. What makes this one special and are there other buttons similarly special? If this is a unique button, then inline the SVG icon as well at least - there's no need to not include it inside the component.

Reply:
I think this is unique case so moving it to gridview makes more sense for me

Reply:
Defined in original gridview with styled.button since there is no need to overlap existing styledcomponent or base one.